savoirfairelinux/num2words
# Implemented (optional) gender-assignment for the spanish language. Sp…

…ecifically, when  users select the ordinal option, they can specify if they want the female assignment (however, male is default since its used for cardinals exclusively (but different for ordinals)).

## Fixes # by... Cole Johnson

### Changes proposed in this pull request:

* Implemented gender option (as parameter) within num2words function.
* The gender is automatically set to male, because it's only when you select ordinals within the Spanish language that you start to consider different gender-assignments
* if char is changed to 'f', then the gender assignment will change to 'a' in the Spanish language (results verified with Spanish dictionary).

### Status

- [ X] READY
- [] WIP (Work-In-Progress)

### How to verify this change

*If you call num2words, and set char='f', and make sure that spanish language/ordinals are true, then you can verify with a Spanish dictionary that the female assignment works.*

### Additional notes

*Worked flawlessly with num2words builds.*
